### DISCUSSION TITLE
incorrect x:class property when creating UserControl

### DISCUSSION THREAD

**Discussion by Kuprich** [AvaloniaUI/Avalonia#13356] [Q&A]
hi everyone

avalonia is amazing project!!!

ide: Visual studio 2022.
I created a new project using "Avalonia MVVM application"
Next step: in folder "View" mouse right click -> add new Item. then i select avalonia user control and press "Add" button.

In this UserControl property "x:Class="TodoAvalonia.UserControl1" but location this Control -  TodoAvalonia.Views.UserControl1"

as a result, x:class property is not defined correctly.

But may be I'am doing something wrong?

**Comment by thevortexcloud** [comment]
Please show your folder structure with the files.

**Reply by Kuprich** [reply]
Hello. I tried to create a similar project on another computer at home. But I didn’t find any errors.
This problem is probably local. I'll try to reinstall the IDE. 
Thank you!

**Comment by Kuprich** [comment]
Local problem with the software on the my computer

**Reply by FBENH** [reply]
How did you solve it? I am having the exact same problem.

**Comment by FBENH** [comment]
I encountered the same problem. While I couldn’t resolve the issue that prevents the view from being created automatically with the correct `x:Class`, I found a workaround. By changing the namespace in the `code-behind` to `AppName.Views` and updating the`x:Class` value to `AppName.Views.YourView`, I was able to fix it.

**Reply by StuRamsey** [reply]
I'm having the same issue and getting around it the same way. My VS2022 is up to date and Avalonia is on 11.2.4, surely we can't have the same PC issue?
